packages/@aws-cdk/aws-appsync/test/integ.appsync-lambda.ts

```diff
@@ -72,6 +72,7 @@ lambdaDS.createResolver({
   fieldName: 'relatedPosts',
   requestMappingTemplate: appsync.MappingTemplate.lambdaRequest(requestPayload('relatedPosts', { withSource: true }), 'BatchInvoke'),
   responseMappingTemplate,
+  maxBatchSize: 5,
```

Comment:
Having cases for a property being specified or not seems like the way to go for testing.

Reply:
I will create another test without the maxBatchSize set.